resources/assets/js/annotations/utils.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * This function checks for invalid annotation shapes.
+ * 
+ * @param feature containing the video annotation to check
+ * @returns true, if a video annotation has an invalid shape, otherwise false.
+ * 
+ * **/
+let isInvalidShape = function (feature) {
+    let geometry = feature.getGeometry();
+    let points = [];
+    switch (geometry.getType()) {
+        case 'Circle':
+            return parseInt(geometry.getRadius()) === 0;
+        case 'LineString':
+            points = geometry.getCoordinates();
+            return (new Set(points.map(xy => String([xy])))).size < 2;
+        case 'Rectangle':
+        case 'Ellipse':
+            points = geometry.getCoordinates()[0];
+            return (new Set(points.map(xy => String([xy])))).size !== 4;
+        case 'Polygon':
+            points = geometry.getCoordinates()[0];
+            return (new Set(points.map(xy => String([xy])))).size < 3;
```

Comment:
Maybe this also has to be:
```suggestion
            return (new Set(points.map(xy => String([xy])))).size < 4;
```
(see above?)

Reply:
At this point, it's right. The points array just saves the coordinates without adding the last one to the end. So a line would consist of only two points and not three.

Reply:
In my test this wasn't the case. When I draw a triangle polygon, I get 4 points here.

Reply:
Ah, I see. My argumentation was wrong here. Sorry. 

If I apply this change, I cannot draw valid triangle polygons any more. 
There are 4 coordinates, where the first one is duplicated and added to the end. But the number of distinct coordinates still remains by at least 3. Your change would allow only rectangle polygons.